flight/search/cell.py

```diff
@@ -0,0 +1,34 @@
+"""
+Summary
+-------
+defines the Cell dataclass
```

Comment:
Don't need a header for this.
```suggestion
defines the Cell dataclass
```